examples/pytorch/jknet/model.py

```diff
@@ -0,0 +1,62 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import dgl.function as fn
+from dgl.nn.pytorch.conv import GraphConv
+
+class JKNet(nn.Module):
+    def __init__(self,
+                 in_dim,
+                 hid_dim,
+                 out_dim,
+                 num_layers=1,
+                 mode='cat',
+                 dropout=0.):
+        super(JKNet, self).__init__()
+        
+        self.mode = mode
+        self.dropout = nn.Dropout(dropout)
+        self.layers = nn.ModuleList()
+        self.layers.append(GraphConv(in_dim, hid_dim, activation=F.relu))
```

Comment:
Have you tried different options of `norm` for `GraphConv`?

Reply:
The results were not much different when `norm` was set as 'both' or 'right', but became worse when 'none'.